From: SeongJae Park <sj@kernel.org>
Date: Tue, 18 Mar 2025 11:35:32 -0700
Subject: [PATCH] ==== [PATCH 0/2] mm/damon: introduce DAMOS filter type for
 active pages ====

[PATCH 0/2] mm/damon: introduce DAMOS filter type for active pages

The memory reclaim algorithm categorizes pages into active and inactive
lists, separately for file and anon pages. The system's performance
relies heavily on the (relative and absolute) accuracy of this
categorization.

This patch series add a new DAMOS filter for pages' activeness, giving
us visibility into the access frequency of the pages on each list. This
insight can help us diagnose issues with the active-inactive balancing
dynamics, and make decisions to optimize reclaim efficiency and memory
utilization.

For instance, we might decide to enable DAMON_LRU_SORT, if we find that
there are pages on the active list that are infrequently accessed, or
less frequently accessed than pages on the inactive list.

Nhat Pham (2):
  mm/damon: implement a new DAMOS filter type for active pages
  Docs/mm/damon/design: document active DAMOS filter type

 Documentation/mm/damon/design.rst | 2 ++
 include/linux/damon.h             | 2 ++
 mm/damon/paddr.c                  | 3 +++
 mm/damon/sysfs-schemes.c          | 1 +
 4 files changed, 8 insertions(+)

base-commit: 40caf747267c18b6206e26a37d6ea6b695236c11
--
2.47.1

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_0t9mtmhc | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_0t9mtmhc

diff --git a/hkml_cv_bogus/hkml_cv_bogus_0t9mtmhc b/hkml_cv_bogus/hkml_cv_bogus_0t9mtmhc
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

